haas/api.py

```diff
@@ -534,6 +530,16 @@ def headnode_detach_network(headnode, hnic):
                             # Network Code #
                             ################
 
+@rest_call('GET', '/networks')
+def list_networks():
+    """List all networks.
+    Returns a JSON array of strings representing a list of projects.
+    Example:  '["network1", "network2", "network3"]'
+    """
+    db = model.Session()
+    networks = db.query(model.Network).all()
+    networks = [n.label for n in networks]
+    return json.dumps(networks, sort_keys=True)
```

Comment:
Would it be possible to add an optional "verbose" here, which would also show you the vlan ID for each network (which is shown by `show_network()`)?